yanic/yanic.conf.tmpl

```diff
@@ -59,7 +59,7 @@ path = "/srv/yanic/meshviewer-ffdon.json"
 
 [nodes.output.meshviewer-ffrgb.filter]
 sites = ["ffdon"]
-
+domain_as_site = true
```

Comment:
This is how I got it to work now, and with the `[respondd.sites.ffdon]` section above being removed:
```suggestion
domain_as_site = true
sites = [{% for domain in sites["ffdon"] %}"ffdon_{{ domain }}", {% endfor %}]
```